<?xml version="1.0"?>
<launch>


  <include file="$(find panda_hand_eye_calibrate)/launch/ar_marker.launch">
    <arg name="marker_size" value="10.0" />
    <arg name="cam_image_topic" value="/kinect2/qhd/image_color" />
    <arg name="cam_info_topic" value="/kinect2/qhd/camera_info" />
    <arg name="output_frame" value="/kinect2_rgb_optical_frame" />
  </include>



  <include file="$(find easy_handeye)/launch/calibrate.launch">
    <arg name="eye_on_hand" value="false"/>
    <arg name="namespace_prefix" value="panda_eob_calib"/>

    <!-- fill in the following parameters according to your robot's published tf frames -->
    <arg name="robot_base_frame" value="panda_link0"/>
    <arg name="robot_effector_frame" value="panda_EE"/>

    <!-- fill in the following parameters according to your tracking system's published tf frames -->
    <arg name="tracking_base_frame" value="kinect2_link"/>
    <arg name="tracking_marker_frame" value="ar_marker_7"/>


    <arg name="freehand_robot_movement" value="false" />
    <arg name="robot_velocity_scaling" value="0.5" />
    <arg name="robot_acceleration_scaling" value="0.2" />
    <arg name="start_rviz" value="false" />
    <arg name="start_sampling_gui" value="true" />

    <arg name="move_group" value="panda_arm"  doc="the name of move_group for the automatic robot motion with MoveIt!" />



  </include>
</launch>
